src/utils/stac.ts

```diff
@@ -33,12 +33,19 @@ interface _Contact { // see: https://github.com/stac-extensions/contacts?tab=rea
 
 export type Contact = RequireAtLeastOne<_Contact, "name" | "organization">;
 
-export interface Dimension {
-  type: string;
-  axis?: "x" | "y" | "z";
-  description?: string;
-  unit?: string;
-}
+export type Dimension =
+  | {
```

Comment:
This or (`|`) seems odd...
```suggestion
  {
```